healthtools/tests/test_scrapers.py

```diff
@@ -68,9 +69,21 @@ def test_doctors_scraper_archives_to_s3(self):
         uploaded_data = self.doctors_scraper.s3.get_object(
             Bucket="cfa-healthtools-ke",
             Key=self.doctors_scraper.s3_key
-        )['Body'].read()
+            )['Body'].read()
         self.assertEqual(uploaded_data, data)
 
+    # get test/health_facilities.json key for this test
+    # def test_health_facilities_scraper_archives_to_s3(self):
+    #     self.health_facilities_scraper.s3_key = "test/health_facilities.json"
+    #     with open(TEST_DIR + "/dummy_files/health_facilities.json", "r") as my_file:
+    #         data = my_file.read()
+    #         self.health_facilities_scraper.archive_data(data)
+    #     uploaded_data = self.health_facilities_scraper.s3.get_object(
+    #         Bucket="cfa-healthtools-ke",
+    #         Key=self.health_facilities_scraper.s3_key
+    #         )['Body'].read()
+    #     self.assertEqual(uploaded_data, data)
```

Comment:
I tried writing tests for health facilities and test that they are uploading but it was failing because of the key. Funny enough I removed it too in the other branch